frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -207,11 +210,13 @@ export function DashCardVisualization({
 
     return (
       <DashCardMenuConnected
+        downloadsEnabled={downloadsEnabled}
         question={question}
         result={mainSeries}
         dashcardId={dashcard.id}
         dashboardId={dashboard.id}
         token={isEmbed ? String(dashcard.dashboard_id) : undefined}
+        uuid={isUuid(dashcard.dashboard_id) ? dashcard.dashboard_id : undefined}
```

Comment:
This is used by the download function to decide which endpoint to use, it was not used before for public dashboard/questions so it was not needed

Reply:
Correct me if I'm wrong, this prop already exists, but we don't currently use it before this PR?
https://github.com/metabase/metabase/blob/cad5e3f7f290aa83ddceb561063c31e97ad7ddbc/frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu.tsx#L61

Reply:
Correct, it was indeed already a prop but it was never passed